src/sentry/static/sentry/app/components/gridEditable/addColumnButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import ToolTip from 'app/components/tooltip';
+
+import {GRID_HEADER_HEIGHT, Z_INDEX_ADD_COLUMN} from './styles';
+
+// this is an even number
+const BUTTON_SIZE = 20;
+const PLUS_SIGN_WIDTH = 2;
+const PLUS_SIGN_HEIGHT = 12;
+
+type Props = {
+  onClick: () => void;
+};
+
+const AddColumnButton = (props: Props) => {
+  const {onClick} = props;
+  return (
+    <Wrapper onClick={onClick}>
+      <ToolTip title={t('Add Column')}>
+        <div style={{width: `${BUTTON_SIZE}px`, height: `${BUTTON_SIZE}px`}}>
+          <Vertical />
+          <Horizontal />
```

Comment:
We should get an SVG icon for this.

Reply:
I'll try asking Dora and John.

Reply:
Or re-use the + in a circle we already have.

Reply:
Ok I'll look into it.

Reply:
Done.